### PR TITLE
Hydrate indicator widget

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- Hydrate indicator widget

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/cms/index.js [hunk 1]
```diff
@@ -162,6 +178,28 @@ export function renderBlocks({
...
     <>
+      {Array.from(
+        document.querySelectorAll(`div[id^=${TYPES.INDICATOR_WIDGET}]`)
+      ).map(el => {
+        // deprecated php indicator used to have innerhtml in its container div
+        // eslint-disable-next-line no-param-reassign
+        el.innerHTML = '';
+        return ReactDOM.createPortal(
+          <Card
+            id={el.getAttribute('id')}
```
Comment (karimkawambwa) [inline src/cms/index.js]: Should this not be `POST_ID` ? If not still make it a new const.

File: src/components/Card/ChartContainer.js [hunk 2]
```diff
@@ -77,12 +77,10 @@ ChartContainer.propTypes = {
   embedCode: propTypes.string,
   dataLinkGeoId: propTypes.string,
   dataLinkTitle: propTypes.string,
-  dataLinkHref: propTypes.oneOfType([propTypes.string, propTypes.func])
-    .isRequired,
+  dataLinkHref: propTypes.oneOfType([propTypes.string, propTypes.func]),
   analysisLinkCountrySlug: propTypes.string,
   analysisLinkTitle: propTypes.string,
-  analysisLinkHref: propTypes.oneOfType([propTypes.string, propTypes.func])
-    .isRequired,
```
Comment (karimkawambwa) [inline src/components/Card/ChartContainer.js]: Good looking out 👍

File: src/core/PDFDataContainer.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+import Pdf from '@mikecousins/react-pdf';
+import LeftArrow from './assets/icons/left-arrow.svg';
+import RightArrow from './assets/icons/right-arrow.svg';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    padding: '0.625rem',
+    display: 'flex',
```
Comment (karimkawambwa) [inline src/core/PDFDataContainer.js]: minor: unnecessary
```suggestion
```

File: src/cms/index.js (hunk 1 above)
Comment (KhadijaMahanga) [reply src/cms/index.js]: Made it a constant. Indicators are not custom post type, so we can't have `POST_ID`

**Pull request merged**